data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,224 @@ mission "Remnant: Visit to Paradise 1"
+
+
+
+mission "Remnant: Shattered Light 2"
+	name "Searching for Ghosts"
+	description "Take Ruach to find and scan the ghostly ship."
+	blocked "Your commlink chimes with a note that this mission requires a Research Laboratory as well as 2 cargo space and 1 free bunk."
+	cargo "equipment" 2
+	passengers 1
+	source "Viminal"
+	to offer
+		has "Remnant: Shattered Light 1: done"
+	npc "scan outfits"
+		government "Ember Waste"
+		personality uninterested waiting staying mute
+		system Levana
+		ship "Dromedary Specter" "???"
+		dialog "This ship seems a bit more solid than it did when you first saw it, but it does not seem to be quite present."
+	on offer
+		require "Research Laboratory"
+		conversation
+			`Your arrival in the starport finds Taely in deep conversation with an elderly Remnant, empty food trays cluttering the table before them. They turn as one when you approach, breaking off their conversation.`
+			`	"Thank you for coming, Captain <last>. I would like to introduce you to Ruach. She was a shipyard engineering prefect for many decades, but now spends most of her time as a teacher." Ruach gives a polite bow. "Taely lays too many compliments on me," she gestures with amusement. "I fulfilled my duty doing the best with that which the Embers blessed me, nothing more, nothing less. But it has been rewarding watching someone take everything I taught them and fly with it to heights I could only dream of."`
+			`	Ruach's gestures take on a more precise cadence. "Taely tells me that you have spotted a strange 'ghost' in Caeculus that bears a resemblance to ancestral ships. Could you take me to do a scan of it please?"`
+			choice
+				`	"OK"`
+					accept
+				`	"Sorry, not interested."`
+					decline
+	on complete
+		payment 50000
+		conversation
+			`The <ship> has not even settled onto the landing pad when Ruach leaps down from the airlock and jogs a couple spots down to stare up at the gills of an Albatross. A few moments later, Taely drops down to the ground beside her. As you approach you can see Ruach has a hologram of the spectral ship floating in the air between them. "It is lacking the confirmation codes or identification info, but I am confident that this is it." Ruach says as you approach.`
+			`	"Sorry for leaving you behind, this is just such an exciting discovery. This ship you have found matches the profile of the Shattered Light - a large city ship that was part of the exodus but never arrived on this side of the wormhole. There are so many questions here! Anyway, the data is being distributed right now. Meet me in the cafeteria in a few hours to see what ideas have come up. Oh, and before I forget, we have allotted you <payment> for your assistance."`
```

Comment:
```suggestion
			`	"Sorry for leaving you behind, but this is just such an exciting discovery. This ship you found matches the profile of the Shattered Light - a large city ship that was part of the exodus but never arrived on this side of the wormhole. There are so many questions to be answered! Anyway, the data is being distributed right now. Meet me in the cafeteria in a few hours to see what ideas have come up. Oh, and before I forget, we have allotted you <payment> for your assistance."`
```

Reply:
I am tempted to add an action before "Anyway," but I'm running a bit tight on time right now, so go figure.